RepositoryCompiler/Startup.cs

```diff
@@ -42,6 +45,21 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             {
                 endpoints.MapControllers();
             });
+
+            app.UseRabbitListener();
+        }
+    }
+
+    public static class ApplicationBuilderExtentions
```

Comment:
Do we understand how this class, and the code below works exactly? We'll need to discuss this on Firday.